### ISSUE TITLE
Application does not start when putting new JSON files in Examples folder

### ISSUE THREAD

**Issue by jacovanderlaan** [RoelantVos/Virtual-Data-Warehouse#9]
Hi Roelant,

I have created 2 JSON files in TEAM (version 1.6.9) which I have put in the Examples folder of VDW.
The files are attached.
[STG_STOCK_PORTAL.zip](https://github.com/RoelantVos/Virtual-Data-Warehouse/files/5584437/STG_STOCK_PORTAL.zip)
After putting the files in this folder VDW does not want to start anymore without giving me an error.
Any advice?

Regards, 

Jaco

**Comment by RoelantVos** [comment]
Hi @jacovanderlaan thanks for this. I have looked into this and the reason is that the releases from TEAM and VDW are slightly differently timed and the most recent VDW release (1.6.2) has been updated to comply with the latest DWH automation schema whereas the most recent release for TEAM (v1.6.0) has not been updated to match this schema version yet.

For example, the TEAM version creates Json files with the segment DataObjectMappingList and this has been changed to DataObjectMappings (along with various other changes as per v1.2. of the DWH automation schema).

In short, it is not possible to use this version of TEAM to create Json files for the most recent release of VDW. Having said that, I will keep this issue open to add better event handling because the user should be informed gracefully of this.

In the meantime, the in-development v1.6.1 of TEAM is all but done and is expected to be released next week or so and is good enough to work with directly if you compile it from the v1.6.1. (development) branch.

**Comment by RoelantVos** [comment]
This is now updated. VDW uses the mapping name, the classification (at Data Object level) to generate the screens. If this information is not available, this is defaulted to the name of the target (for the mapping name) and 'Miscellaneous' for the classification.
